pkg/commands/builder/save_test.go

```diff
@@ -16,7 +16,7 @@ import (
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 
-	"github.com/pivotal/build-service-cli/pkg/commands/builder"
+	bldr "github.com/pivotal/build-service-cli/pkg/commands/builder"
```

Comment:
I think idiomatically you'd rename the variable before the package

Reply:
wow, did not know that... will make that change - package reference as builder and the variable as bldr.

Reply:
addressed in commit https://github.com/vmware-tanzu/kpack-cli/commit/a61ff4ef0a04a510b0d135f156ed9396832cbce6

marking as resolved for now. feel free to reopen if required.